eng/scripts/Language-Settings.ps1

```diff
@@ -157,6 +159,8 @@ function Publish-java-GithubIODocs ($DocLocation, $PublicArtifactLocation)
 }
 
 function Get-java-GithubIoDocIndex() {
+  # Fill in language specific information using script.
+  Mutate-Files -appTitle $AppTitle -lang $Language -indexhtmlloc $IndexHtmlLoc
```

Comment:
Sorry I didn't call this out in your tools PR but a function called `Mutate-Files` is a little to generic and I'm not at all sure what it is doing from this context. We should probably name it something like "UpdateDocIndexFiles" or something like that.

Reply:
... ```Update-DocIndexFiles``` to be more idiomatic for PowerShell ;)

Reply:
I'm fine with `Update-` in this case but in generally I only try to match the verb naming pattern when writing cmdlets not functions, I really hate trying to map my functions to an approved verb so I generally don't :)

Reply:
![image](https://user-images.githubusercontent.com/513398/100868479-c6149d80-34ef-11eb-9048-1cd4f313fa65.png)